DisplayBook.tsx

```diff
@@ -40,13 +56,22 @@ export default class DisplayBook extends Component {
 
 const styles = StyleSheet.create({
     main_container: {
```

Comment:
en javascript on comme convention d'utiliser le camelCase plutot que le l'underscore pour les nom de variables
```suggestion
    mainContainer: {
```